content/docs/command-reference/remove.md

```diff
@@ -13,35 +14,32 @@ positional arguments:
 
 ## Description
 
-This command safely removes stages from
-[dvc.yaml](/doc/user-guide/dvc-files-and-directories#dvcyaml-file), their
-`.gitignore` entries, and optionally removes from the <abbr>workspace</abbr>
-files or directories that are tracked by DVC. It takes one or more stage names
-(see `-n` option of `dvc run`) or `.dvc` files as target, removes it, and
-optionally removes all of its outputs (`outs` field).
+This command can safely remove stages from `dvc.yaml` or `.dvc` files. This
```

Comment:
`sages from .dvc files` - this doesn't sound correct?

Reply:
Just noticed this but it was fixed before. Thanks